scripts/generator/templates/template-release-notes.md

```diff
@@ -2,41 +2,67 @@ Vaadin {{platform}}
 
 [Changelogs](#_changelogs) · [Upgrading guides](#_upgrading_guides) · [Docs](https://vaadin.com/docs/latest/) · [Get Started](https://vaadin.com/start/)
 
-## New and Noteworthy Since Vaadin 23.0
+## New and Noteworthy Since Vaadin 23.2
 
 ### Flow
-- Support for Liferay 7
-- Vite build tool improvements `*`
+- Preserve on refresh: add refresh flag to navigation events
+  _Adds the `isRefreshEvent` flag to `BeforeEnterEvent` and `AfterNavigationEvent` to distinguish if the event is for refresh on the view with `@PreserveOnRefresh`_
+- Better Vite compilation logs
+- Allow setting content of the Html component
+  _Adds `Html::setHtmlContent()` to sets the content based on the given HTML fragment (thanks @TatuLund !)_ 
+- Rerouting to external URL
+  _Redirects to the given URL in `BeforeEnterEvent::forwardToUrl()` even if the servlet is deployed with a non-root context path_
+- Use new `auto-configuration` in spring-boot
+- Auto-update of the tsconfig
+  _Auto-updates `tsconfig.json` in the project and gives a descriptive message to the developer asking to add custom changes manually (if any)_
+- Gradle plugin: Filtering dependencies during prepare frontend task
+  _Speeds up the prepare frontend task execution for the cases when the number of dependencies of the project is very large and the types of dependencies are very different (thanks @mvysny !)_
 
 ### Hilla
-- Reactive Endpoints `*`
-
+- Hilla React
+  - New starter CLI application presets based on React
+  - @hilla/react-components npm package with wrappers of all Vaadin web components (excl. Map and Spreadsheet)
+  - [Documentation](hilla.dev/docs/react/start/basics)
+  
 ### Design System
-- Lit renderer directives
-- Grid: freeze columns to the end
-- Dialog: header and footer slots
-- Lumo: Java API for CSS utility classes
-- Tree Grid: eager fetch mode
-- Multi-Select Combo Box
-
-### Collaboration Engine
-- `FormManager` and `ListOperation` API
-- Clustering support `*`
-
-`*` experimental
+- Stable **Spreadsheet** component for flow
+- New component: **TabSheet**
+- New component: **Tooltip** 
+- Chain validation in inputs: 1. web-component, 2. Flow component, 3. Binder.
+- **DatePicker**: 2-digit year parsing
+- Shift-click for **Grid** multi-sort
+
+### Vaadin Kits
+- Collaboration Engine 
```

Comment:
Rename to Collaboration Kit
Include changes in CE from 5.2.0 to 5.3.0

Reply:
done